docs/_docs/usage.md

```diff
@@ -7,8 +7,8 @@ The Jekyll gem makes a `jekyll` executable available to you in your terminal.
 
 You can use this command in a number of ways:
 
-* `jekyll new` - Creates a new Jekyll site with default gem-based theme
-* `jekyll new --blank` - Creates a new blank Jekyll site scaffold
+* `jekyll new .` - Creates a new Jekyll site in present working directory with default gem-based theme. The `.` can be replaced with a path. The path will be created if it does not exist.
```

Comment:
Please recommend `jekyll new PATH` instead.
`jekyll new .` will throw an error if the current directory is not empty. In that situation, one will need to use `--force`:
```
jekyll new . --force
```

```suggestion
* `jekyll new some/path` - Creates a new Jekyll site with default gem-based theme at `./some/path`. The directories will be created as necessary.
```